src/lib.rs

```diff
@@ -127,6 +157,17 @@ impl PtObjects {
                 Ok((idx, stop_points))
             })
             .collect::<StdResult<BTreeMap<_, _>, Error>>()?;
+        let lines_to_comments =
+            ManyToMany::from_forward(create_relation_with_comments::<Line>(&c.lines));
```

Comment:
@TeXitoi is it necessary to write `::<Line>` ?

Reply:
It shouldn't, but that's easier to just test.

Reply:
:white_check_mark: